src/sagas/deposit.js

```diff
@@ -162,7 +192,13 @@ export function* requestADeposit() {
         deposit = yield call([tbtc.Deposit, tbtc.Deposit.withSatoshiLotSize], new BN(1000000))
     } catch (err) {
         if (err.message.includes(METAMASK_TX_DENIED_ERROR)) return
-        throw err
+        yield put({
+            type: DEPOSIT_REQUEST_ERROR,
+            payload: {
+                error: err.message,
```

Comment:
I do wonder if we may end up with a caught error with no `message`. Can handle it later, though.

May also make sense to unify on `error` vs `err` for these catches, as well. Again, non-blocking.